vars/Debian-11-pg-9.6.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
As postgresql 9.6 has reached EOL, I would remove all references to this release (and it will simplify configs and tasks in this role ;-))